docs/daily/2024_12_16.md

```
@@ -0,0 +1,36 @@
+형상관리(버전관리) 도구
+
+## Repository
+- Remote Repository(원격 저장소)
+- Local Repository(로컬 저장소)
+
+![](https://lh7-rt.googleusercontent.com/slidesz/AGV_vUdDQanqhN0zoDTYqfqgQR8YeT1u6MJrMKlSKGDyDxh9iiz_nHc2JERkAQO0re4vQCKQkQTXiC6nYCRKoNPI8PZEZ2r8J385bPUGIV_3T6r5BaIfiyuJ85w-RSo6VchT-dVbf9rkzQVrdk3wU_lpYNJ8hQ=s2048?key=CnSYR3bHv9n7fmrGx7oyHQ)
+
+
+## 필수 깃 명령어
+- git init
+    - 현재위치에 저장소 만들기
```

Markdownlint (0.37.0)

Expected: 2; Actual: 4
Unordered list indentation

(MD007, ul-indent)